u2flib_server/utils.py

```diff
@@ -56,8 +70,15 @@ def sha_256(data):
     return h.digest()
 
 
-Rand.rand_seed(os.urandom(1024))
+def rand_bytes(n_bytes):
+    return os.urandom(n_bytes)
 
 
-def rand_bytes(n_bytes):
-    return Rand.rand_bytes(n_bytes)
+def verify_ecdsa_signature(payload, pubkey, signature):
+    verifier = pubkey.verifier(signature, ec.ECDSA(hashes.SHA256()))
+    verifier.update(payload)
+
+    try:
+        verifier.verify()
+    except InvalidSignature:
+        raise Exception('Attestation signature verification failed!')
```

Comment:
A 4th option would be to just have the function return a boolean, and skip the exception altogether (but still catching InvalidSignature). I'm not sure if verify() would ever raise a different exception, but if so it would just pass up the chain.